engines/chamber/kult.cpp

```diff
@@ -175,7 +173,9 @@ void gameLoop(byte *target) {
 
 			updateUndrawCursor(target);
 			refreshSpritesData();
-			runCommandKeepSp();
+			uint16 restart = runCommandKeepSp();
+			if (restart == 1 && g_vm->_shouldRestart)
```

Comment:
I would avoid direct usage of magic constant `1`. Please use instead something like
```C++
#define RUNCOMMAND_RESTART 1337
```

e.g. set some completely unusual number and assign it to a constant.